platforms/models.py

```diff
@@ -24,3 +24,11 @@ class Word(models.Model):
 
     def __str__(self):
         return self.word
+
+class UsersWords(models.Model):
+    user_word = models.ForeignKey(Word, on_delete=models.CASCADE, related_name='users_words')
+    user_name = models.ForeignKey(User, on_delete=models.CASCADE, related_name='users_words')
+    time = models.DateTimeField(auto_now_add=True)
```

Comment:
1. should not be auto-add. we need to control where we want to fill it.
2. Name , time of what? I cannot understand purpose of this field by the name